postman/cac/custom types/Get Types/event.test.js

```diff
@@ -7,42 +7,41 @@ pm.test('expect response be 200', function () {
     console.log("The API returned the response", modified_response);
     pm.expect(JSON.stringify(modified_response)).to.be.eq(JSON.stringify([
```

Comment:
Why did this change?

Reply:
Had to change the order to pass the test, don't know why was it failing though.